include/net/net_context.h

```diff
@@ -349,41 +363,52 @@ static inline void net_context_set_state(struct net_context *context,
  */
 static inline sa_family_t net_context_get_family(struct net_context *context)
 {
+	int family = (context->flags & NET_CONTEXT_FAMILY) >> 4;
+
 	NET_ASSERT(context);
 
-	if (context->flags & NET_CONTEXT_FAMILY) {
+	if (family == NET_CONTEXT_AF_INET) {
+		return AF_INET;
+	} else if (family == NET_CONTEXT_AF_INET6) {
 		return AF_INET6;
+	} else if (family == NET_CONTEXT_AF_PACKET) {
+		return AF_PACKET;
 	}
 
-	return AF_INET;
+	return AF_UNSPEC;
 }
 
 /**
  * @brief Set address family for this network context.
  *
- * @details This function sets the address family (IPv4 or IPv6)
+ * @details This function sets the address family (IPv4, IPv6, packet)
```

Comment:
> This function sets the address family (IPv4, IPv6, packet)

"packet address family" sounds strange to me. Suggestion: write "(IPv4, IPV6, etc.)" or remove that part altogether.

Reply:
What is strange with AF_PACKET? No need to change anything here.

Reply:
Which AF_PACKET? There's no "AF_PACKET", there's "packet", and it's strange, yeah.